### DISCUSSION TITLE
Say Hallo

### DISCUSSION THREAD

**Discussion by SilasK** [metagenome-atlas/Tutorial_Finnland#4] [General]
Hey I'm Silas Kieser, From Switzerland and I'm looking forward to having this course with you. 

For more information about me https://silask.github.io

**Comment by S0R0N** [comment]
Hi all. 
I am Santiago Ruiz Navas, From Colombia; I like dogs and dancing, and I am looking forward to learning with and from all of you.

**Reply by SilasK** [reply]
Did you get the link for the Gather town?

**Comment by gafnialmog** [comment]
Hello,

My name is Almog, I am from Israel. 
Currently a postdoc at the Ben Gurion University studing Environmental Microbiology at the Environmental Earth department. 
Hope I can follow up :)

I really appreciate the opportunity.

Thank you

**Comment by myeong-nuri** [comment]
Hi all :D
I'm Nuri, from the Republic of Korea and I'm pursuing a Doctoral in Microbial Ecology at Korea Polar Research Institution.

Thank you for your time.

**Comment by Sewunet-Abera** [comment]
Hello all,
I'm Sewunet from Ethiopia. I'm currently in the Netherlands (NIOO-KNAW, Leiden University) doing my phd on plant rhizosphere microbiome compositional and functional shits in response to a parasitic weed attack. 
Thank you for having me here.

**Comment by Lehosmaa** [comment]
Hello all, 
I am Kaisa Lehosmaa, I work as postdoctoral researcher in Anna Maria Pirttilä's host-microbe research group (University of Oulu, Finland). If you have any problems during the course (e.g., log in to CSC etc.) you can also contact me. Thank you Silas for teaching/organising atlas course - I am looking forward for the course!

**Comment by XinTan0814** [comment]
Hello everyone,
I am Xin Tan, from China. I am a doctoral candidate in Harbin Institute of Technology.
Thanks very much to the course lecturer and the organizers of this course. I couldn’t wait to participate this course.

**Comment by yanxianl** [comment]
Hello everybody,
I'm Yanxian Li from China. I'm currently a phd candidate at Nowegian University of Life Sciences, working on aquaculture, fish nutrition, gut health and microbiota. 
Thanks Silas and the sponsors for organizing this great metagenomic workshop. Looking forwards to a fun learning experience ahead!

**Comment by nganguyen89** [comment]
Hello, 
I am Nga Nguyen, a PhD student of University of Oulu, Finland. I am currently working on bilberry fruit ripening. 
I am looking forward to learning from you all.
Thank you!

**Comment by beatencio** [comment]
Hi everyone!
I'm Betzabe Atencio from Panama. Currently, I am a Ph.D. student at Ben Gurion University of the Negev in Israel, studying microbial communities (taxonomic and functional diversity) in groundwater.  
Thanks for organizing this course and the opportunity!
Betza

**Comment by KRMustonen** [comment]
Hello everyone,

I am Kaisa-Riikka Mustonen from the University of Oulu, Finland. I am working as a postdoctoral researcher in Arctic Ecology and Biogeochemistry research group led my Prof. Jeffrey Welker. I am excited to learn new things with you all!

**Comment by jmalazar** [comment]
Hi everyone! 

My name is Jacqueline Malazarte doing a PhD at the University of Oulu under Prof. Timo Muotka. My research is about the dynamics of microbial communities in a river network and how they respond to environmental change. 

Looking forward to learn and meet new people. 😃

**Comment by giosang1111** [comment]
Hi, I'm Phuong Nguyen, a student at the University of Oulu. I study endophytic communities in bilberry fruits.
Nice to meet you all.

**Comment by huuskok** [comment]
Hi all,
I’m Karoliina Huusko. I am working as a postdoctoral researcher at the Ecology and Genetics at the University of Oulu. I study plant-microbial interactions, soil and root microbial communities and diversity at various geographical scales in natural and human-modified environments. Thanks for organizing this course!

**Comment by SaijaA** [comment]
Hello,
I'm Saija Ahonen, a PhD student at the University of Oulu, Finland. My thesis focuses on tundra soil microbial communities but I have been also working with many other types of environmental samples.
Thanks for organizing the course and I'm looking forward to learn new things :)

**Comment by Stefan-L** [comment]
Hi everyone!
I'm Stefan Lambert, I am postdoc at the University of Oulu. I work with Katharina Kujala on bioremedation of heavy metals in peatlands in Lapland. Thanks for organizing this course, I'm looking forward to learning new things about Atlas!

**Comment by katharinakujala** [comment]
Hello everyone,
I'm Katharina Kujala and I'm coordinating a small group working on environmental microbiology at the University of Oulu. I have never used ATLAS, but I have worked a little with metagenomes. I'm looking forward to a great course.

**Comment by ilma74** [comment]
Hi,
I am Ilma Tapio, I am researcher at Natural Resources Institute Finland studying microbial communities associated with ruminants. I am looking forward to learn more about metagenome data analyses.

**Comment by slambrechts** [comment]
Hi everyone,
I'm Sam Lambrechts, and I am currently working with Antarctic soil samples at the University of Ghent. Thank you for organizing this course, I'm looking forward to it.

**Comment by jmtsuji** [comment]
Hi all,

Nice to meet you! I'm Jackson Tsuji, a postdoctoral fellow in Prof. Manabu Fukui's research group at Hokkaido University in Japan. I did my PhD in Prof. Josh Neufeld's lab at the University of Waterloo in Canada. My main focus these days is on ["_Candidatus_ Chlorohelix allophototropha"](https://www.biorxiv.org/content/10.1101/2020.07.07.190934v3), a beautiful and highly unusual filamentous phototroph that could hold new clues to understanding the origin of oxygenic photosynthesis. During my PhD, I worked a lot with ATLAS to tease apart phototrophic microbial communities in iron-rich Boreal Shield lakes, and I continue to use ATLAS on a wide variety of projects. I'm excited to join you on Oct. 4th as a guest speaker to share some of my experiences using ATLAS in microbial ecology research.

**Comment by binutripathi** [comment]
Hi everyone,
I'm Binu Tripathi and I am a postdoctoral researcher at Korea Polar Research Institute. I am a microbial ecologist studying soil microbiome in various terrestrial ecosystems. Thank you for organizing this interesting course! I'm looking forward to meet new people and learn new skills in metagenome data analysis.
Thank you!
Binu

**Reply by srisvs33** [reply]
Hello Everyone, My self Venkatachalam Siddarthan, currently working as a post-doctoral fellow at the National Centre for Polar and Ocean Research, India.  I am from a Microbial Ecology background and my current research activities are mainly focused on Arctic fjords. I look forward to interacting with everyone. Cheers, Venkat

**Comment by lyonskvn** [comment]
Hallo! My name is Kevin Lyons. Originally from Ireland, but currently working as a doctoral researcher at the University of Oulu studying groundwater microbes. I have some experience with 16S rRNA amplicon sequencing analysis, but not much with metagenomes. Looking forward to the course. Thanks for organising! :)

**Comment by maudimi** [comment]
Hello everyone, my name is Mauricio and I am joining from The Netherlands. Looking forward to learning more about metagenomics and ATLAS, as well as to meet new people. Thanks @SilasK for organising this course. Best Mauricio

**Comment by AmityaDive** [comment]
Hello, my name is Amit. Thanks Silas for organizing this workshop!
